src/__tests__/helpers/utils.js

```diff
@@ -37,14 +37,21 @@ function addEventListener(el, type, listener, options) {
   el.addEventListener(type, hijackedListener, options)
 }
 
-function setup(ui) {
-  const {
+function setup(ui, {shadowRootSelector} = {}) {
+  let hostElement
+  let {
     container: {firstChild: element},
   } = render(ui)
+
+  if (shadowRootSelector) {
+    hostElement = element
+    element = element.shadowRoot.querySelector(shadowRootSelector)
+  }
+
   element.previousTestData = getTestData(element)
 
   const {getEventCalls, clearEventCalls} = addListeners(element)
-  return {element, getEventCalls, clearEventCalls}
+  return {element, hostElement, getEventCalls, clearEventCalls}
 }
```

Comment:
Rather than complicating the `setup` function (especially for what will likely be only a very small number of tests that need it), let's just not use the setup function in the test and instead use `addListeners`.

Reply:
yea good idea! fixing...